domino/domino.py

```diff
@@ -281,11 +281,14 @@ def collaborators_add(self, usernameOrEmail, message=""):
             return disposition
 
     # App functions
-    def app_publish(self, unpublishRunningApps=True):
+    def app_publish(self, unpublishRunningApps=True, hardwareTierId=None):
         if unpublishRunningApps is True:
             self.app_unpublish()
         url = self._routes.app_publish()
-        request = {"language": "App"}
+        if hardwareTierId is None:
+            request = {"language": "App"}
+        else:
+            request = {"language": "App", "hardwareTierId": hardwareTierId}
```

Comment:
Is this if/else unnecessary? Doesn't the API behave the same way either way, since `hardwareTierId` is implicitly `None` if you don't pass it in?

Reply:
from looking elsewhere in this file I think you are correct. I think I can just write:
```
request = {"language": "App", "hardwareTierId": hardwareTierId}
```

thanks

Reply:
updated.